demo-app/src/open-truss/configs/2-available-workflows.yaml

```diff
@@ -0,0 +1,10 @@
+workflow:
+  version: 1
+  frames:
+    - frame:
+      view:
+        component: AvailableWorkflowsFromEndpoint
```

Comment:
@jonmagic @kmcq when you have a moment I would appreciate your help debugging an issue I'm seeing.

`AvailableWorkflowsFromEndpoint` is declared in the demo-app and `OTAvailableWorkflowsFromEndpoint` is declared in the package. They are pretty much identical, but I'm running into a react error for the OT version. Ya'll are much better versed in React than me, so any help would be appreciated!

![Model Home 2024-01-11 at 4 47 44 PM](https://github.com/open-truss/open-truss/assets/4481584/98ff5009-f9fc-4852-aa4d-c052bac0523a)


![オートメーション 2024-01-11 at 4 47 11 PM](https://github.com/open-truss/open-truss/assets/4481584/06f3c5ed-1459-4074-a289-10564cba2a46)

Reply:
NOTE: I merged a branch with some version changes into main and then merged main into your branch here.

I'm not able to reproduce this error, what url are you accessing? I tried the following urls and they load fine:

- http://localhost:3000/ot/playground
- http://localhost:3000/ot/playground/2-available-workflows

Reply:
Ah, you need to update the config to use `OTAvailableWorkflowsFromEndpoint` and not `AvailableWorkflowsFromEndpoint`. That will update it to use the component that is in the package instead of the demo-app.